CMakeLists.txt

```diff
@@ -74,15 +74,18 @@ option(use_prov_client "Enable provisioning client" OFF)
 option(use_tpm_simulator "tpm simulator type of hsm used with the provisioning client" OFF)
 option(use_edge_modules "Enable support for running modules against Azure IoT Edge" OFF)
 option(use_custom_heap "use externally defined heap functions instead of the malloc family" OFF)
-option(use_uhttp_upload "set use_uhttp_upload to ON to use uhttp for upload to blob" OFF)
+option(use_uhttp_upload "set use_uhttp_upload to ON to use uhttp for upload to blob" ON)
+option(use_uhttp_upload_logging "set use_uhttp_upload_logging to ON to enable uhttp blob logging (only when use_uhttp_upload is also ON)" OFF)
```

Comment:
This seems a little off. There shouldn't be an cmake option for uhttp in the azure-iot-SDK-c

Reply:
None of these are for uhttp - both are for the SDKs:
- first one is for using uhttp for blob upload (IoT SDK + BLOB storage SDK).
- second one is to control logging for the BLOB SDK.